### ISSUE TITLE
Starting loop

### ISSUE THREAD

**Issue by Bfender** [ETCDEVTeam/emerald-wallet#522]
After some days with 0.8.1 RC 2 installed, done it some withdraws to the wallet, and today the wallet don't start.
It shows the next windows and don't change.
OS: Win 7
Any help ?
Thank you in advance.
![image](https://user-images.githubusercontent.com/37271938/37256254-4c6ecb4e-2558-11e8-922d-eb540904d7f2.png)

**Comment by mylonelyplanet** [comment]
same issue encountered

**Comment by h3110w0r1d** [comment]
me too

**Comment by tipsntrix** [comment]
Me too... Help

**Comment by splix** [comment]
One of remote endpoints was under maintenance/upgrade, so some users had this issue. That remove is upgraded now afaik, so it should start working.

Anyway, #523 will allow to switch to another network if there will be a similar issue in future

**Comment by Bfender** [comment]
Sorry, but it continues with the same problem and same error.
Any help ?
I don't understand how to apply the changes in main.js file, I don't finded it.
Can I remove the wallet and refresh with a new install, I am afraid that if I do this, I will lose my ETC deposit on wallet.
Thank you.

**Comment by Bfender** [comment]
Thank you @splix , removing setting file it worked.
I have recovered the wallet with file key.
Now I can see my ETC deposit again on wallet.

**Comment by splix** [comment]
hm, if I understand correctly after removing settings file you missed your account in address list so you had to import it from backup? that shouldn't be a case. can you confirm that deleting setting file removed accounts?

**Comment by Bfender** [comment]
Sorry, I am a newbie here.
I have tried again to remove the settings files and when I restart the wallet, the steps are:
1.- User agreement
2.- Select full node
3.-Open wallet
4.- When main window appears change from Mainnet to Mainnet (gastracker.io )
5.-After a little synchronizing, I can see my deposit this time.

So this time I have not missed my account.
Perhaps is a posibility that before I have removed too a folder named ".emerald" on Roaming folder ( Windows 7 )
